### PR TITLE
Update airlift to 249

### PR DESCRIPTION
**Pull request by wendigo**
## Description

Brings proper support for HTTP/2 on the server side.

Part of https://github.com/trinodb/trino/issues/22271

Fixes https://github.com/trinodb/trino/issues/21793

## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: core/trino-main/pom.xml [hunk 1]
```diff
@@ -415,6 +415,12 @@
             <groupId>io.airlift</groupId>
             <artifactId>jaxrs-testing</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
```
Comment (losipiuk) [inline core/trino-main/pom.xml:419]: comment why needed and move exclusion to airlift if this is problematic and not needed transitive dep

**Review by losipiuk — APPROVED** [review]
% comments

File: core/trino-main/pom.xml (hunk 1 above)
Comment (wendigo) [reply core/trino-main/pom.xml:419]: There are two hamcrest versions on classpath with different GAV so I cannot settle a single version: [org.hamcrest:hamcrest-core:1.3, org.hamcrest:hamcrest:2.2]

**Comment by mosabua** [comment]
This needs a release notes entry @wendigo

**Pull request merged**